### PR TITLE
ADD filter func on zoom event

### PR DESCRIPTION
**Pull request by sarovin**
Hi,
I need the filter func on zoom event for filter some gesture.
For example:
```
const filter = useCallback((d3Event) => {
    return d3Event.ctrlKey
  }, []);

...
<ZoomableGroup zoom={1} filterZoomEvent={filter}  >
...
```

SEE d3-zoom doc => https://github.com/d3/d3-zoom#zoom_filter

### CODE REVIEW

**Comment by sarovin** [comment]
@zimrick please, this is an important feature for us.

File: src/components/useZoomPan.js [hunk 1]
```diff
@@ -56,7 +57,17 @@ export default function useZoomPan({
       onMoveEnd({ coordinates: [x, y], zoom: d3Event.transform.k }, d3Event)
     }
 
+    function filterFunc() {
+      if (filter) {
```
Comment (zimrick) [inline src/components/useZoomPan.js:61]: There doesn't seem to be any definition of `filter` anywhere. Did you mean to check for `filterZoomEvent` here?

Comment (sarovin) [reply src/components/useZoomPan.js]: Yes, I am sorry.
I have fixed it.

**Comment by zimrick** [comment]
Added in [v2.1.0](https://github.com/zcreativelabs/react-simple-maps/releases/tag/v2.1.0)

**Pull request merged**